modules/ROOT/pages/single-sign-on-with-kerberos.adoc

```diff
@@ -39,13 +39,13 @@ image:images/kerberos-overview.png[Authentication over Kerberos]
 
 The scenario covered by the filter is the following:
 
-* The Bonita Kerberos filter intercepts all the requests to Bonita portal pages, and checks if the user is already logged in on Bonita
+* The Bonita Kerberos filter intercepts all the requests to Bonita Application pages, and checks if the user is already logged in on Bonita
  ** If already logged in \=> Allow the access
  ** If not logged in \=> The request is transferred to the Spnego authenticator
 
 The Spnego authenticator will then verify the user's Kerberos tickets if present or create a new one. After obtaining a valid ticket the filter will store some information (the authenticated user login) in the client request and get back to the Bonita Kerberos filter.
 
-* The Bonita Kerberos filter will automatically create a Bonita session and let the user through to access the Portal resources.
+* The Bonita Kerberos filter will automatically create a Bonita session and let the user through to access the Bonita Application resources.
```

Comment:
The Bonita Kerberos filter will automatically create a Bonita session and let the user through to access the Bonita Application resources. => The Bonita Kerberos filter will automatically create a Bonita session and let the user access the Bonita Application resources.

Reply:
Done